ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -26,22 +26,25 @@
           description: "A pull request ID, like 'origin/pr/72/head'"
 
     triggers:
+      - reverse:
```

Comment:
I've never seen this before, could you explain a bit what is going on with this?

Reply:
Sure! 
(Found this in the documentation:
https://docs.openstack.org/infra/jenkins-job-builder/triggers.html?highlight=triggers#triggers.reverse)

So my thought here was if make check passes based on a (dashboard-related) PR we could trigger the ceph-dashboard-pull-requests Jenkins job.
But what I'm not sure/wondering about here is if this will now trigger it for every PR or if it will only trigger the Job if the ceph-pull-requests Job passed AND the dashboard label I've defined in line 36/37 is assigned to the PR.

Reply:
yeah, I think this will cause this to run for both almost all the time. I would just leave it as an ad-hoc call to trigger the tests. This is how we do it for ceph-volume: we don't need to run on every PR, just on a selected few